api/src/main/java/org/pmiops/workbench/cdr/CdrVersionService.java

```diff
@@ -62,7 +62,7 @@ public void setCdrVersion(DbCdrVersion version) {
     }
 
     String authorizationDomain = version.getAccessTier().getAuthDomainName();
-    if (!fireCloudService.isUserMemberOfGroup(
+    if (!fireCloudService.isUserMemberOfGroupWithCache(
```

Comment:
This is another interactive-only usage

Reply:
Thoughts on this given request-scoped comment above^ ?